x-pack/auditbeat/module/system/package/package.go

```diff
@@ -41,6 +41,7 @@ const (
 
 	bucketName              = "package.v1"
 	bucketKeyPackages       = "packages"
+	bucketKeyPackagesFb     = "packages_fb"
```

Comment:
so this was done intentionally to move over existing users from gob encoded storage to flat buffer encoded storage. The most efficient way I found for this migration was to use a new bucket key instead of checking everytime if the packages are gob encoded or not. So people upgrading to the new version, on their 1st savePackagesToDisk op will have their old packages migrated to the new encoding stored with the new key and the old bucket key will be removed.

Reply:
This is a reference to https://github.com/elastic/beats/pull/34817/files/1cffa0172972f6cd9182161416b00501c97433dc#diff-e1b7a2865862ac4361d442cb09ccd6987196ce7b8028b9b67d9d279054125d7a